src/lib/components/css/table.css

```diff
@@ -0,0 +1,73 @@
+table:not(.table) {
```

Comment:
These `:not(.table)` modifiers are there temporarily until we fully migrate from legacy theme